assets/constants.ts

```diff
@@ -7041,6 +7041,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "show-translation-contributions",
+  "SHORT_COMMIT_HASH": "52e6c6798"
```

Comment:
Revert this, I know these changes are not done by you and it is happening automatically.
Thanks